zcl-builtin/matter/zcl.json

```diff
@@ -197,5 +197,8 @@
     ]
   },
   "defaultReportingPolicy": "mandatory",
-  "ZCLDataTypes": ["ARRAY", "BITMAP", "ENUM", "NUMBER", "STRING", "STRUCT"]
+  "ZCLDataTypes": ["ARRAY", "BITMAP", "ENUM", "NUMBER", "STRING", "STRUCT"],
+  "uiOptions": {
+    "showProfileId": false
```

Comment:
This is how it should be yes, but UI at some point then turns this into "isZigbee". I don't understand that.